src/components/bottom_bar/bottom_bar.xml

```diff
@@ -2,23 +2,27 @@
   <t t-name="o-spreadsheet-BottomBar">
     <div
       class="o-spreadsheet-bottom-bar o-two-columns d-flex align-items-center overflow-hidden"
+      t-att-class="{'ps-3 flex-fill': env.isSmall}"
```

Comment:
Any way to have the isSmall stuff handled in the css rather than in a bunch of t-att-class in the xml ?